pkg/release/release.go

```diff
@@ -108,29 +171,81 @@ func IsDirtyBuild(build string) bool {
 	return strings.Contains(build, "dirty")
 }
 
+// TODO: Consider collapsing some of these functions.
+//       Keeping them as-is for now as kubepkg is dependent on them.
+func GetStableReleaseKubeVersion() (string, error) {
+	logrus.Info("Retrieving Kubernetes release version...")
+	return GetKubeVersion("https://dl.k8s.io/release/stable.txt")
+}
+
+func GetStablePrereleaseKubeVersion() (string, error) {
+	logrus.Info("Retrieving Kubernetes testing version...")
+	return GetKubeVersion("https://dl.k8s.io/release/latest.txt")
+}
+
+func GetLatestCIKubeVersion() (string, error) {
+	logrus.Info("Retrieving Kubernetes latest build version...")
+	return GetKubeVersion("https://dl.k8s.io/ci/latest.txt")
```

Comment:
We have three different urls here. I think it will be better to put them as const at the beginning of the file. It will be more future proof if the links will be used somewhere else, and also more consise.